pkg/gateway/lb/handler/grpc/pairing_heap_test.go

```diff
@@ -0,0 +1,518 @@
+// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package grpc
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/test/goleak"
+)
+
+// NOT IMPLEMENTED BELOW
```

Comment:
Is it going to be implemented?

Reply:
Regarding the Test task for LB Handler, overall it has not been started, so I think it would be better to do it all together when testing this package.
Also, it is necessary to include the aggregation_test.go, etc.

For the verification of this additional code, I think that E2E Stream and BenchmarkCode of this package are enough to verify the operation, so I don't think that adding this code in a hurry is necessary.

Reply:
Overall understood. LGTM for this PR.

Some extra notes. You can ignore it.

- I was imagining that the pairing heap algorithm was isolated and could be tested independently of the LB.
- In my understanding, benchmarks are for ensuring execution speed, and are not for verifying the operation itself.

Reply:
It would be better to write a test for the ordering of the PairingHeap.

On the other hand, there are multiple methods for the PairingHeap, and writing tests for each method would require a lot of man-hours and should be done at another time.